js/scripts.js

```diff
@@ -0,0 +1,14 @@
+
+$(function(){
+	var span = $('span'); 
+	span.each(function(index, element) {
+		if(index % 2 == 0) { $("span:even").css('color', 'red'); 
```

Comment:
to co jest w środku tej funkcji, wykona się dla każdego elementu. `span:even` to wybieranie wszystkich parzystych elementów z całego dokumentu. Więc jak będziesz miał 10000 spanów, to w tym loopie 10000 razy zaznaczysz 5000 spanów na kolor czerwony. trochę niepotrzebnie. skoro już przechodzisz przez każdy span w dokumencie (span.each) to w środku odwołuj się do pojedynczego spana (który jest dostępny pod `element`)